src/scripts/Save.ts

```diff
@@ -226,7 +226,7 @@ class Save {
     public static convert() {
         const base64 = $('#convertTextArea').val().toString();
         try {
-            const json = atob(base64);
+            const json = decodeURI(atob(base64));
```

Comment:
This shouldn't be needed, `convert` is only used to import shinies from the legacy pokeclicker version, which won't have your uri encoding.